apps/server/src/modules/sharing/uc/share-token.uc.spec.ts

```diff
@@ -71,20 +109,18 @@ describe('ShareTokenUC', () => {
 			});
 
 			it('should not check any other permissions', async () => {
```

Comment:
title does not fit that well with the code of the test:
from: it should not check any other permissions
to__: it should check permissionByReference once

Reply:
But it's old code... so don't bother.